app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -30,8 +30,17 @@ def verify_credentials(_auth_type = nil, options = {})
         # As the connect will only give a handle
         # we verify the credentials via an actual operation
         connect(options).inventory.list_feeds
+      rescue URI::InvalidComponentError
+        raise MiqException::MiqHostError, "Host '#{hostname}' is invalid"
```

Comment:
That would happen when hostname is:
- https://localhost
- localhost:8080

Hostname should only be 'localhost' as port and the protocol are being handled appart.